circuits/circom/updateStateTree.circom

```diff
@@ -35,10 +32,10 @@ template UpdateStateTree(
     // Output: New state tree root
     signal output root;
 
-    // Input(s)
+    // The coordinator's public key
     signal input coordinator_public_key[2];
 
-    // Note: a message is an encrypted command
+    // Note that a message is an encrypted command
     var message_length = 11;
```

Comment:
nitpick: Should we capitalize `message_length`, `message_signature_length`, and `message_without_signature_length` too?

Reply:
Absolutely! Thanks for spotting this!